microk8s-resources/actions/common/containerd-config-migrate.py

```diff
@@ -0,0 +1,71 @@
+import shutil
```

Comment:
@balchua, is the migration needed? Containerd says it is backwards compatible with the previous version, does this include the config it runs with? I think It runs ok with the old config.

Reply:
I couldn't get the gpu working with the old config.

Reply:
I think gpu was not working because we did not change the runtime in the initial commit.

I did some testing and it seems fine to me now. Let me know what you think.

Reply:
You mean the `$RUNTIME`?

Reply:
> You mean the `$RUNTIME`?

Yes